packages/generator/test/prisma/field.test.ts

```diff
@@ -6,20 +6,18 @@ describe("Field model", () => {
     expect(Field.parse("userId: int").toString()).toMatchInlineSnapshot(`"userId   int"`)
   })
 
+  it("generates models for pluralized property, making pluralized name fields", () => {
+    expect(Field.parse("names:string").toString()).toMatchInlineSnapshot(`"names  String"`)
+    expect(Field.parse("userIds: int").toString()).toMatchInlineSnapshot(`"userIds   int"`)
+  })
+
   it("serializes optional types", () => {
     expect(Field.parse("name:string?").toString()).toMatchInlineSnapshot(`"name  String?"`)
   })
 
   it("serializes list types, pluralizing fields", () => {
     expect(Field.parse("users:int[]").toString()).toMatchInlineSnapshot(`"users  Int[]"`)
-    expect(Field.parse("user:int[]").toString()).toMatchInlineSnapshot(`"users  Int[]"`)
-  })
-
-  it("pluralizes lists and makes single fields singular", () => {
-    expect(Field.parse("name:string").toString()).toEqual(Field.parse("names:string").toString())
-    expect(Field.parse("userIds:string[]").toString()).toEqual(
-      Field.parse("userId:string[]").toString(),
-    )
+    expect(Field.parse("user:int[]").toString()).toMatchInlineSnapshot(`"user  Int[]"`)
```

Comment:
Prevent field name from being converted to plural forms when creating a Field with a list type property.